src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,32 @@
+export class PokemonClient {
+  async fetchPokemons(ids) {
+    const allPromises = [];
+    ids.forEach((id) => {
+      allPromises.push(fetch(`https://pokeapi.co/api/v2/pokemon/${id}/`));
+    });
+
+    try {
+      let AllResponses = await Promise.all(allPromises);
+      AllResponses = [...AllResponses.map((response) => response.json())];
+      let AllData = await Promise.all(AllResponses);
```

Comment:
Im not sure I understand why do you need promise all here. All responses should be an array of the parsed responses.

Reply:
then how can I resolve the response to get the pokemon object?